Remove reference to SL for Claude MCP setup (#8508)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/integrate-mcp-claude.md

To configure Claude Desktop to use the dbt MCP server:
1. Go to the [latest dbt MCP release](https://github.com/dbt-labs/dbt-mcp/releases/latest) and download the `dbt-mcp.mcpb` file.
2. Double-click the downloaded file to open it in Claude Desktop.

> 3. Configure the **dbt Platform Host**. You can find this in your <Constant name="dbt_platform" /> account by navigating to Dashboard, then Settings.

Reviewer: @DevonFulcher , i found this hard to understand bc the locations didn't match dbt platform. does 'dbt platform host' mean the 'access urls'? e.g. `https://access_url.us1.dbt.com/`?

<img width="1197" height="393" alt="Image" src="https://github.com/user-attachments/assets/4d8f8484-f7e1-4b9f-9e82-1d4fd8e02249" />

```suggestion
3. Configure the **dbt Platform Host**. You can find this in your <Constant name="dbt_platform" /> account by navigating to **Account settings** and copying the **Access URL**. 
```

Author: Yes, that is it. Thanks!
